php/php.editor/src/org/netbeans/modules/php/editor/csl/SemanticAnalysis.java

```diff
@@ -727,11 +742,17 @@ public void visit(ConstantDeclaration node) {
                 parentNode = path.get(1);
             }
             if (parentNode instanceof ClassDeclaration || parentNode instanceof InterfaceDeclaration
-                    || parentNode instanceof TraitDeclaration) {
+                    || parentNode instanceof TraitDeclaration || parentNode instanceof ClassInstanceCreation) {
+                boolean isPrivate = Modifier.isPrivate(node.getModifier());
                 List<Identifier> names = node.getNames();
                 if (!names.isEmpty()) {
```

Comment:
Although this is existing code, we can delete this check.

Reply:
Removed.